Assets/MixedRealityToolkit/Utilities/Rendering/DepthBufferRenderer.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Rendering
+{
+    [ExecuteInEditMode]
+    public class DepthBufferRenderer : MonoBehaviour
+    {
+        public RenderTexture outputTexture;
```

Comment:
Please update this to a serialized private field with a public accessor if needed

https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#encapsulation